src/Compiler/Checking/MethodCalls.fs

```diff
@@ -1120,7 +1120,7 @@ let TryImportProvidedMethodBaseAsLibraryIntrinsic (amap: Import.ImportMap, m: ra
     match tryTcrefOfAppTy amap.g declaringType with
     | ValueSome declaringEntity ->
         if not declaringEntity.IsLocalRef && ccuEq declaringEntity.nlr.Ccu amap.g.fslibCcu then
-            let n = mbase.PUntaint((fun x -> x.GetParameters().Length), m)
+            let n = mbase.PApplyArray((fun x -> x.GetParameters()),"GetParameters", m).Length
```

Comment:
nameof for "GetParameters"?

````
> let x = typeof<int>;;
    nameof x.ToString;;
val it: string = "ToString"
````

Reply:
No access to an instance to apply `nameof` to. mbase is a `tainted`, so cannot really reach it's symbols :(